### PR TITLE
Add checksum validation for Netherlands

### PR DESCRIPTION
**Pull request by sh4dowb**
#159 
@mdomke

### CODE REVIEW

File: schwifty/checksum/netherlands.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from schwifty import registry
+
+
+register = functools.partial(checksum.register, prefix="NL")
+
+
+@register
+class DefaultAlgorithm(checksum.Algorithm):
+    name = "default"
+    accepts = checksum.InputType.BBAN
```
Comment (mdomke) [inline schwifty/checksum/netherlands.py:13]: If you're actually just interested in the account code, you can also use `InputType.ACCOUNT_CODE`.

File: schwifty/checksum/netherlands.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    def compute(self, bban: str) -> str:
+        raise Exception('Cannot compute checksum for Netherlands')
+
+    def validate(self, bban: str) -> bool:
+        acc_number = bban[-10:]
+        total = 0
+        for i, digit in enumerate(map(int, acc_number)):
+            total += i * (10-i)
```
Comment (mdomke) [inline schwifty/checksum/netherlands.py:22]: Are you certain that this is implementation is correct? It only uses the index variable `i` and never makes use of the `digit` at the corresponding position. Do you maybe have some reference documentation for the algorithm itself?

File: schwifty/checksum/netherlands.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import functools
+
+from schwifty import checksum
+from schwifty import registry
```
Comment (mdomke) [inline schwifty/checksum/netherlands.py:4]: This is imported but never used.

File: schwifty/checksum/netherlands.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+register = functools.partial(checksum.register, prefix="NL")
+
+
+@register
+class DefaultAlgorithm(checksum.Algorithm):
+    name = "default"
+    accepts = checksum.InputType.BBAN
+
+    def compute(self, bban: str) -> str:
+        raise Exception('Cannot compute checksum for Netherlands')
```
Comment (mdomke) [inline schwifty/checksum/netherlands.py:16]: How about raising a `NotImplementedError`?

File: schwifty/checksum/netherlands.py (hunk 2 above)
Comment (sh4dowb) [reply schwifty/checksum/netherlands.py:22]: damn, yeah sorry, my mistake.. should be `total += digit * (10-i)`

I wrote this a while ago, don't remember the source though, I remember it took me a long time googling
```
# Netherlands account number algorithm uses this factor table:
# digit | factor
# 1st   | 10
# 2nd   | 9
# ...
# 10th  | 1
# 
# After multiplying digits according to the table, sum of these results must be divisible with 11
```

File: schwifty/checksum/netherlands.py (hunk 1 above)
Comment (sh4dowb) [reply schwifty/checksum/netherlands.py:13]: for this and others, I'd be glad if you could edit it, I'm not very familiar with this codebase, just had the "rare" NL algo and wanted to help out. I copied BE checksum file and edited that.

**Review by mdomke — APPROVED** [review]

**Pull request merged**